# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatformMac.cpp
# Commit: bf0dd96b4423
# Full Hash: bf0dd96b442327acd6df0db0f80886a53a4cec52
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 15:40:28
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 7 - Route COLRv1 fonts on macOS through our COLR-rendering path; only COLRv0 can use the native Core Text support. r=gfx-reviewers,lsalzman
#   
#   Depends on D153880
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152162
# ==============================================================================

diff -r d0a5d11277bb -r bf0dd96b4423 gfx/thebes/gfxPlatformMac.cpp
--- a/gfx/thebes/gfxPlatformMac.cpp	Wed Aug 17 10:59:35 2022 +0000
+++ b/gfx/thebes/gfxPlatformMac.cpp	Wed Aug 17 10:59:35 2022 +0000
@@ -142,10 +142,6 @@
   mFontAntiAliasingThreshold = ReadAntiAliasingThreshold();
 
   InitBackendPrefs(GetBackendPrefs());
-
-  if (nsCocoaFeatures::OnHighSierraOrLater()) {
-    mHasNativeColrFontSupport = true;
-  }
 }
 
 gfxPlatformMac::~gfxPlatformMac() { gfxCoreTextShaper::Shutdown(); }
